### PR TITLE
Sean / Update DP2P LP content

### PR DESCRIPTION
**Pull request by sean-binary**
Changes:

-   Update DP2P LP content

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [x] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: src/pages/p2p/components/_numbers.js [hunk 1]
```diff
@@ -8,11 +8,11 @@ import device from 'themes/device'
 
 const content = [
     {
-        header: <Localize translate_text="2000+" />,
+        header: <Localize translate_text="4000" />,
```
Comment (AbdallaMourad) [inline src/pages/p2p/components/_numbers.js:11]: I'm just curious is there a translation for numbers 😄

**Review by AbdallaMourad — APPROVED** [review]
LGMT

**Review by yashim-deriv — APPROVED** [review]

File: src/pages/p2p/components/_numbers.js (hunk 1 above)
Comment (sean-binary) [reply src/pages/p2p/components/_numbers.js:11]: Good question, I mean there is but I don't know for this instance whether they would want to translate it to non-number figures 😂

Comment (AbdallaMourad) [reply src/pages/p2p/components/_numbers.js:11]: I see, makes sense 😄

File: src/pages/p2p/components/_dp2p.js [hunk 2]
```diff
@@ -12,12 +12,16 @@ const StyledSection = styled(SectionContainer)`
     padding: 8rem 0 0 0;
 
     @media ${device.tabletL} {
-        padding-top: 40px;
+        padding: 40px 16px 0 16px;
```
Comment (AbdallaMourad) [inline src/pages/p2p/components/_dp2p.js:15]:
```suggestion
        padding: 40px 16px 0;
```

**Pull request merged**